### PR TITLE
Upgrade Unicode stuff from version 11.0.0 to 13.0.0.

### PR DESCRIPTION
**Pull request by magnumripper**

### CODE REVIEW

File: doc/NEWS [hunk 1]
```diff
@@ -117,6 +117,11 @@ Major changes from 1.9.0-jumbo-1 (May 2019) in this bleeding-edge version:
   used before compression.  Before this we had false negatives for those.
   Also updated LZMA code from upstream, to versioj 19.00.
 
+- Upgraded Unicode stuff from version 11.0.0 to 13.0.0 (this affects the
+  external modes Dumb16, Dumb32, Repeats16 and Repeats32.  It also affects
+  the eastern-egg option "--subsets=full-unicode" which BTW supersedes the
+  external modes with far better performance.
```
Comment (solardiz) [inline doc/NEWS]: Missed closing brace for "(this affects ...".

For the eastern-egg option, I think we should mention it in comments near the external mode definitions. I think we should similarly mention the "--subsets" option in general near the Subsets external mode definitions.

**Review by solardiz — APPROVED** [review]
Good stuff. Please feel free to fix the NEWS/comments or just merge as-is.

**Comment by solardiz** [comment]
BTW, I've been using "rebase and merge" lately, as "squash and merge" lists GitHub (and not the person) as the committer in git history.

File: doc/NEWS (hunk 1 above)
Comment (magnumripper) [reply doc/NEWS]: Thanks, I'll fix that soon.

Comment (solardiz) [reply doc/NEWS]: Also, s/eastern-egg/Easter egg/ and you need to add your nickname and year like we've been doing on other NEWS entries.

**Comment by magnumripper** [comment]
> BTW, I've been using "rebase and merge" lately, as "squash and merge" lists GitHub (and not the person) as the committer in git history.

Yeah, I always used "rebase and merge" while I'd do any squashes manually/locally (preferably by the author) and force-pushing to the topic branch.

BTW there's some fine command-line git logic for tagging commits for later fixup/squash, I learnt it first from https://thoughtbot.com/blog/autosquashing-git-commits and have used it a lot periodically. I **think** auto-squash can even be used within the GitHub web GUI, at least with some add-on stuff - but I never tried that (actually I'm not interested in replacing quick and exact command-line stuff with fuzzy GUI stuff - I have a feeling you'd agree).

**Comment by magnumripper** [comment]
Done. I also added missing attribution of the previous NEWS item in a separate commit but kept it in this PR

**Pull request merged**